client/app/bundles/shared/components/Checkbox.jsx

```diff
@@ -0,0 +1,46 @@
+// @flow
+import React from 'react';
+import css from './Checkbox.scss';
+
+type Props = {
+  id?: string,
+  checked?: boolean
+};
+
+type State = {
+  checked: boolean;
+};
+
+export default class Checkbox extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = { checked: this.props.checked || false };
+  }
+
+  toggle = () => {
+    this.setState({ checked: !this.state.checked });
+  }
+
+  render() {
+    const checkedOrNot = [];
+    checkedOrNot.push(
+      <p>{this.state.checked ? 'Unchecked' : 'Checked'}</p>,
+    );
+    const { id } = this.props;
+
+    return (
+      <span>
+        <form>
+          <label htmlFor={id} className={css.switch} >Option1
+            <input type="checkbox" tabIndex="0" onChange={this.toggle} />
+            <div className={css.check}>{}</div>
+          </label>
+          <label htmlFor={id} className={css.switch}>Option2
```

Comment:
Let's get rid of this second checkbox, since the component should just be one radio button. If we want multiple on a page then we will do `<Checkbox etc. />` multiple times.